### DISCUSSION TITLE
What's the vision for v2.0?

### DISCUSSION THREAD

**Discussion by sklibanov312** [zauberzeug/nicegui#2486] [General]
There's a v2.0 branch here that looks to be active.

Is there a document that describes the vision for how that branch will be different from v1 and an approximate timeline for maturity?

**Comment by rodja** [comment]
In 2.0 we are currently only accumulating smaller breaking API changes. There are no new earth shattering changes planned right now. We are very happy with the current architecture. To be honest also quite surprised that it turned out so well and gradual improvements are still the best way forward.

**Comment by falkoschindler** [comment]
Note that version 2.0 is basically a 1.5, but we want to switch to [semantic versioning](https://semver.org/). So we increase the major version because of breaking changes to the API.
